edb/edgeql/compiler/context.py

```diff
@@ -232,6 +232,9 @@ class Environment:
     compiled_stmts: Dict[qlast.Statement, irast.Stmt]
     """A mapping of from input edgeql to compiled IR"""
 
+    actually_toplevel_result_view_name: Optional[s_name.QualName]
```

Comment:
Drop `actually`.

Reply:
I'm going with `alias_result_view_name`. I don't want it to just have the same name as `toplevel_result_view_name` in context, which is different